helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -78,6 +78,10 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.agent.podSecurityContext }}
```

Comment:
I see `$agent.securityContext` used in this file quite a bit, please take a look?

Reply:
`$agent` is to support the old section that was called `otelAgent`. It's not relevant for new fields

Reply:
`podSecurityContext` is a new field